shell/server/drivers/mail.js

```diff
@@ -260,11 +267,12 @@ hackSendEmail = function (session, email) {
       });
     }
 
-    if (!(this.userId in dailySendCounts)) {
-      dailySendCounts[this.userId] = 0;
-    }
-    var sentToday = ++dailySendCounts[this.userId];
-    if (sentToday > DAILY_LIMIT) {
+    var user = Meteor.users.findAndModify({
```

Comment:
By default, findAndModify returns the old document (see http://docs.mongodb.org/manual/reference/command/findAndModify/). It's not a huge deal, but technically people can send one email over their DAILY_LIMIT. You can add `new: true` to the `findAndModify` or change the `if` statement, if you care.